config.yaml

```diff
@@ -270,11 +270,13 @@
             id: syn18689335
             format: csv
           - name: proteomics_tmt
-            id: syn33344657
-            format: json
+            id: syn35221005
+            format: csv
         final_format: json
         custom_transformations: 1
         provenance:
           - syn18689335
-          - syn33344657
+          - syn35221005
+        column_rename:
+          log_2fc: log2_fc
```

Comment:
We don't need to rename a column to the same name, so this can be removed. This applies to test_config.yaml too.